lib/screen/dailies_work/dailies_work_page.dart

```diff
@@ -82,14 +92,53 @@ class DailyWorkPageState extends State<DailyWorkPage>
     super.dispose();
   }
 
-  void _pageControllerListenser() {
+  void _pageControllerListener() {
     if (_pageController!.page != 0) {
       pauseDailyWork();
     } else {
       resumeDailyWork();
     }
   }
 
+  void openDisplayDialog() {
+    // ignore isSubscribed because it's not necessary
+    unawaited(_displayButtonKey.currentState?.onTap(context, true));
+  }
+
+  void didPushed() {
+    _stopTrackingInterest();
+  }
+
+  void trackInterest() {
+    if (_trackingInterest) {
+      return;
+    }
+    log.info('start trackingInterest in Daily');
+    _trackingInterest = true;
+    _trackingInterestTimer = Timer(_stayDurationLikingThreshold, () {
+      _setUserInterested();
+    });
+  }
+
+  void _stopTrackingInterest() {
+    log.info('stopTrackingInterest in Daily');
+    _trackingInterest = false;
+    _trackingInterestTimer?.cancel();
+  }
+
+  void _setUserInterested() {
+    if (!_trackingInterest) {
+      return;
+    }
+    log.info('Set User Interested in Daily');
+    _stopTrackingInterest();
```

Comment:
could you please call this after _setUserInterested

Reply:
why? then i have to call it 3 times